funfact/algorithm.py

```diff
@@ -92,10 +92,10 @@ class _Factorization(Factorization, ab.AutoGradMixin):
                 'funfact.loss.'
             )
     try:
-        loss(target, target, **kwargs)
+        loss(target, target, append=append, **kwargs)
     except Exception as e:
         raise AssertionError(
-            f'The given loss function does not accept two arguments:\n{e}'
+            f'The given loss function does not accept three arguments:\n{e}'
```

Comment:
The 'two' arguments here refer to the target and approximate tensors that the loss function is meant to compare. Any additional arguments are 'meta/hyper-parameters' and hence does not count towards the number of arguments that a loss function accepts. 😁

Reply:
Changed back.